### PR TITLE
Feature - Weather App

### PR DESCRIPTION
**Pull request by cgcarlosg**
At this pull request, I create an app that looks for information on the Open Weather API and shows the temperature, weather description.
Use async, await, and promises
Fetch cities from a JSON file and search for cities that begin with the characters searched

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "webpack-demo",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (trekab) [inline package.json]: Please add the necessary run commands like `build`, `development` to your `package.json`. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+  const days = ['Sun', 'Mon', 'Tues', 'Wed', 'thurs', 'Fri', 'Sat'];
+
+  const day = days[d.getDay()];
+  const date = d.getDate();
+  const month = months[d.getMonth()];
+  const year = d.getFullYear();
+
+  return `${day} ${date} ${month} ${year}`;
+}
```
Comment (trekab) [inline src/index.js]: Make use of ES6 arrow functions for your function declarations. On top of being a mandatory requirement, arrow functions make it easy to follow best practices like enforcing function declaration before function calls. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

**Review by trekab — CHANGES_REQUESTED** [review]
#### Changes Required! :recycle: 
Hi @cgcarlosg! Well done so far! There are some changes for you to make to improve your project though. These include:
- [ ] Add a toggle control to determine if the result will be shown in Celsius or Fahrenheit. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [ ] Please implement error handling for your asynchronous code. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
- [ ] Your `dist` folder is filled with unnecessary files e.g. `screenshot.png`. Please use `webpack` to manage your project's resources and have perhaps have only the `index.html` and `bundle.js` files in your `dist` folder. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/webpack_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
- [ ] More feedback is included in the comments in the code below.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑🛑**

Hello @cgcarlosg ,
This is a great project. You are on the right path. 👏👏
However let's make some changes.

[REQUIRED]: As requested by the previous TSE, kindly add a toggle button to change from celsius to fahrenheit and back.

[REQUIRED]: As requested by the previous TSE, in index.js, line 72, use ES6 by implementing arrow function instead of function keyword.

That's it. Make the required changes and ask for another review👌👌

**Comment by cgcarlosg** [comment]
Good morning
The button is already there, but the live link was not correct, it was fixed.
The keyword function was changed.  
Thanks

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @cgcarlosg :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**